mojo-executor/pom.xml

```diff
@@ -28,21 +28,41 @@
     <name>Mojo Executor</name>
 
     <dependencies>
+        <!-- if you use this outside a Maven plugin the following transitive dependencies need to be provided, otherwise they are provided by the Maven Core classloader  -->
         <dependency>
             <groupId>org.apache.maven</groupId>
             <artifactId>maven-core</artifactId>
+            <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>org.apache.maven</groupId>
             <artifactId>maven-model</artifactId>
+            <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>org.apache.maven</groupId>
             <artifactId>maven-plugin-api</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.codehaus.plexus</groupId>
+            <artifactId>plexus-utils</artifactId>
+            <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>org.codehaus.plexus</groupId>
             <artifactId>plexus-utils</artifactId>
+            <scope>compile</scope><!-- use compile scope due to https://issues.apache.org/jira/browse/MNG-6965  -->
+        </dependency>
```

Comment:
The `plexus-utils` dependency is declared twice here.

Reply:
good catch, fixed in https://github.com/mojo-executor/mojo-executor/pull/75/commits/69f42b6f403db911bd7de50d64dee405f8393a0d